sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ServiceBusReactorSession.java

```diff
@@ -66,14 +66,14 @@
 
     @Override
     public Mono<AmqpReceiveLink> createConsumer(String linkName, String entityPath, MessagingEntityType entityType,
-        Duration timeout, AmqpRetryPolicy retry, ReceiveMode receiveMode, boolean isSession) {
+        Duration timeout, AmqpRetryPolicy retry, ReceiveMode receiveMode, boolean isSession, String sessionId) {
```

Comment:
I'd be inclined to create another method createConsumer and remove the isSession. One createconsumer for normal receives, one for sessions, and another one to support transactions in the future.